index.js

```diff
@@ -3122,19 +3226,16 @@
   //. See also [`not`](#not).
   //.
   //. ```javascript
-  //. > Number.isInteger(42)
+  //. > Number.isInteger (42)
   //. true
   //.
-  //. > S.complement(Number.isInteger, 42)
+  //. > S.complement (Number.isInteger) (42)
   //. false
   //. ```
-  function complement(pred, x) {
-    return !pred(x);
-  }
   _.complement = {
     consts: {},
-    types: [$.Predicate(a), a, $.Boolean],
-    impl: complement
+    types: [$.Predicate (a), a, $.Boolean],
+    impl: compose (not)
```

Comment:
Curried `compose` is very useful!

Reply:
I have been saying the same thing! This is one of the reasons why Ramda's compose is so crippled, but you don't really realise until you try it.